processor/patch.go

```diff
@@ -0,0 +1,70 @@
+package processor
+
+import (
+	"bytes"
+	"go/ast"
+	"go/format"
+	"go/parser"
+	"go/token"
+)
+
+type patch struct {
+	pos   int
+	stmts []ast.Stmt
+}
+
+func (p *Processor) patchFile(fset *token.FileSet, file *ast.File, patches ...patch) error {
+	src, err := formatNode(fset, file)
+	if err != nil {
+		return err
+	}
+
+	var buf bytes.Buffer
+
+	offset := -int(file.FileStart) + 1
```

Comment:
why it is negative though? better for clarity keep it positive and substract when you need relative.
```suggestion
	offsetOriginalFile := int(file.FileStart) + 1
```

Reply:
offset changes during the process, here it is the starting point.
In most cases it is always 1

Reply:
yeah, really need to make this positive. negative file offset is just triggering security alarms bells..